librustzcash/src/rustzcash.rs

```diff
@@ -6,12 +6,10 @@ extern crate libc;
 extern crate pairing;
 extern crate rand;
 extern crate sapling_crypto;
+extern crate zcash_primitives;
 extern crate zcash_proofs;
 extern crate zip32;
 
-mod hashreader;
-
-#[macro_use]
 extern crate lazy_static;
```

Comment:
Are we using anything from `lazy_static` now in the `librustzcash` crate?

Reply:
We call `lazy_static::initialize(&JUBJUB)` in `librustzcash_init_zksnark_params()`. If there's a better place to do this, e.g. as an explicit initialization function in `zcash_primitives`, then we could remove this dependency.